cli/blockchain_info/interfaces.py

```diff
@@ -0,0 +1,26 @@
+"""
+Provide implementation of the blockchain info interfaces.
+"""
+
+
+class BlockchainInfoInterface:
```

Comment:
The same I said in the first message. `BlockchainInfo` > `Batch`. Please, fix this anywhere: folder names, files names, classes, variables, etc. except obviously library usage.

Reply:
Done.